src/Microsoft.Owin.Security.ActiveDirectory/IssuerSigningKeys.cs

```diff
@@ -19,6 +19,6 @@ internal class IssuerSigningKeys
         /// <summary>
         /// Signing tokens.
         /// </summary>
-        public IEnumerable<X509SecurityToken> Tokens { get; set; }
+        public IEnumerable<SecurityKey> Keys { get; set; }
```

Comment:
File an issue here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet